crates/grpc/src/uopool.rs

```diff
@@ -60,7 +78,7 @@ where
 
         let res = {
```

Comment:
The res here should be `Result<T,E>` type. Should I consider the error situation here and return Error status when validation failed?

Reply:
Not sure what you mean here. The error is handled a little bit later (match): https://github.com/Vid201/aa-bundler/pull/162/files#diff-24e5e1943959520a80d3685779d0161bf51ac80c4855df204fefc53db2c6497bR92

Reply:
I see. Why not handled immediately?

Reply:
For `uo_pool.validate_user_operation` we first get immutable reference `uo_pool` and we can have multiple of them. When adding user operation, we need to get mutable which can be only one. That's the reason to split validating and adding. Does this make sense?

Reply:
I get your concern now. The thing I wrote is not related to the problem, I'll make a change.